server/block/cactus.go

```diff
@@ -0,0 +1,106 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/block/model"
+	"github.com/df-mc/dragonfly/server/entity"
+	"github.com/df-mc/dragonfly/server/entity/damage"
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/df-mc/dragonfly/server/world/particle"
+	"github.com/go-gl/mathgl/mgl64"
+	"math/rand"
+)
+
+// Cactus is a naturally occurring block found in deserts
+type Cactus struct {
+	solid
```

Comment:
```suggestion
```

Reply:
following the pattern of wood_stairs, I replaced it with transparent and let the model() over it.